tests/functional/adapter/utils/test_timestamps.py

```diff
@@ -11,7 +11,7 @@ def models(self):
 
     @pytest.fixture(scope="class")
     def expected_schema(self):
-        return {"current_timestamp": "timestamp"}
+        return {"current_timestamp": "timestamp with time zone"}
```

Comment:
Also add a breaking change for time zone types. If users have used types returned by `get_columns_in_relation`, they would have to change their code to accommodate for this.

Reply:
added